packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -5,32 +5,38 @@ import { DevidedPagePath } from '@growi/core';
 import { useTranslation } from 'react-i18next';
 import { UncontrolledTooltip, DropdownToggle } from 'reactstrap';
 
+import { unbookmark } from '~/client/services/page-operation';
 import { toastError } from '~/client/util/apiNotification';
 import { apiv3Get } from '~/client/util/apiv3-client';
 import { IPageHasId } from '~/interfaces/page';
 import { useCurrentUser, useIsGuestUser } from '~/stores/context';
 import loggerFactory from '~/utils/logger';
 
-
 import { MenuItemType, PageItemControl } from '../Common/Dropdown/PageItemControl';
 
 const logger = loggerFactory('growi:BookmarkList');
-
 // TODO: Remove pagination and apply  scrolling (not infinity)
 const ACTIVE_PAGE = 1;
 
 type Props = {
-  pages: IPageHasId[]
+  pages: IPageHasId[] | undefined,
+  refreshBookmarkList: () => void
 }
 
 const BookmarksItem = (props: Props) => {
-  const { pages } = props;
+  const { pages, refreshBookmarkList } = props;
 
-  const generateBookmarkedPageList = pages.map((page) => {
+  const generateBookmarkedPageList = pages?.map((page) => {
     const dPagePath = new DevidedPagePath(page.path, false, true);
     const { latter: pageTitle, former: formerPagePath } = dPagePath;
     const bookmarkItemId = `bookmark-item-${page._id}`;
 
+    const bookmarkMenuItemClickHandler = (async() => {
+      await unbookmark(page._id);
+      refreshBookmarkList();
```

Comment:
`refreshBookmarkList()` (mutate function of SWR)  should be executed on Bookmarks component not inside of this BookmarksItem component .